### PR TITLE
Adds the PR Author to the list of people who can merge

### PR DESCRIPTION
**Pull request by orta**
Re: https://github.com/DefinitelyTyped/dt-mergebot/issues/159

The test fixtures shows the PR would have the merge mutation 👍

### CODE REVIEW

**Comment by orta** [comment]
Interesting, when I commented in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/46229 it merged the PR - perhaps the bot is missing webhooks since we got it running synchronously?

File: src/pr-info.ts [hunk 1]
```diff
@@ -256,7 +256,7 @@ export async function deriveStateForPR(
         headCommitAbbrOid: headCommit.abbreviatedOid,
         headCommitOid: headCommit.oid,
         mergeIsRequested: !!prInfo.comments.nodes
-            && usersSayReadyToMerge(noNulls(prInfo.comments.nodes),
+            && usersSayReadyToMerge(prInfo, noNulls(prInfo.comments.nodes),
                                     dangerLevel.startsWith("Scoped") ? [author, ...allOwners] : [author]),
```
Comment (elibarzilay) [inline src/pr-info.ts:260]: This is not the problem: note that `author` is always included in the list...  I'll re-fetch the fixture for this pr and will try to edit away recent interactions to see if it fails.

Comment (orta) [reply src/pr-info.ts:260]: Cool, 👍  then it is likely that the webhook wasn't being picked up

**Comment by orta** [comment]
This isn't it

**Pull request closed without merging**